server/chat-plugins/chatlog.ts

```diff
@@ -208,7 +208,10 @@ export const LogViewer = new class {
 		buf += `</div>`;
 		if (day !== LogReader.today()) {
 			const nextDay = LogReader.nextDay(day);
-			buf += `<p><a roomid="view-chatlog-${roomid}--${nextDay}" class="blocklink" style="text-align:center">${nextDay}<br />▼</a></p>`;
+			buf += (
+				`<p><a roomid="view-chatlog-${roomid}--${nextDay}${opts ? `--${opts}` : ''}" class="blocklink" ` +
+				`style="text-align:center">${nextDay}<br />▼</a></p>`
+			);
```

Comment:
```suggestion
			const roomid = `view-chatlog-${roomid}--${nextDay}${opts ? `--${opts}` : ''}`;
			buf += `<p><a roomid="${roomid}" class="blocklink" style="text-align:center">${nextDay}<br />▼</a></p>`;
```

You don't actually need to split this into multiple lines (the line length limit doesn't apply to strings) but if you wanted to, this is how I'd do it for readability.

As I've mentioned in the past, I don't like when line breaks break something like this.